documentation/redirects/datasheets.csv

```diff
@@ -38,3 +38,4 @@
 /documentation/hardware/computemodule/dt-blob-dualcam.dts,https://datasheets.raspberrypi.org/cmio/dt-blob-dualcam.dts
 /documentation/hardware/computemodule/example1-overlay.dts,https://datasheets.raspberrypi.org/cm/example1-overlay.dts
 /documentation/hardware/computemodule/minimal-cm-dt-blob.dts,https://datasheets.raspberrypi.org/cm/minimal-cm-dt-blob.dts
+/documentation/linux/software/libcamera/rpi_SOFT_libcamera_1p0.pdf,https://datasheets.raspberrypi.org/camera/raspberry-pi-camera-guide.pdf
```

Comment:
Huh, I wonder how this one got missed out?

I guess we probably ought to add redirects for all 3 versions? https://github.com/raspberrypi/documentation/tree/9f2bf3eec3f554018ee05f839eb6e383e0b86f3c/linux/software/libcamera
i.e. `rpi_SOFT_libcamera_1p0.pdf`, `rpi_SOFT_libcamera_1p1.pdf` & `rpi_SOFT_libcamera_1p2.pdf` ?

Reply:
There could well be others. It was getting fairly frantic near the end of the week. *sighs*